kingofthehill.conf

```diff
@@ -14,6 +14,7 @@ Engine2 = ./stockfish
 EPDBook = ./books/kingofthehill.epd
 BaseTime = 1000
 IncTime = 50
+Move Overhead = 1000
```

Comment:
I think this would simply be ignored, because I can not find code in `spsa.pl` that uses this variable.